src/app/users/users-profile/users-profile.component.html

```diff
@@ -9,7 +9,7 @@
     <span>{{userDetail.name}}</span>
     <mat-icon class="margin-lr-5" *ngIf="userDetail.gender" svgIcon="{{userDetail.gender}}"></mat-icon>
     <span class="toolbar-fill"></span>
-    <button mat-stroked-button class="margin-lr-3" *ngIf="(user.isUserAdmin && hasAchievement)" [routerLink]="['../../achievements/', urlName, { planet: userDetail.planetCode }]"><mat-icon class="margin-lr-3"s>visibility</mat-icon><span i18n>View Achievements</span></button>
+    <button mat-stroked-button class="margin-lr-3" *ngIf="hasAchievement" [routerLink]="['../../achievements/', urlName, { planet: userDetail.planetCode }]"><mat-icon class="margin-lr-3"s>visibility</mat-icon><span i18n>View Achievements</span></button>
```

Comment:
it needs to be either manager or users own profile. Right now this changes allows to see achievement of any user by anyone.